src/source/statsd.rs

```diff
@@ -25,10 +26,35 @@ pub struct Statsd {
     host: String,
     port: u16,
     tags: sync::Arc<metric::TagMap>,
+    parse_config: sync::Arc<StatsdParseConfig>,
+}
+
+/// The mask type for metrics in `StatsdParseConfig`.
+pub type Mask = Regex;
+
+/// The bound type for metrics in `StatsdParseConfig`.
+pub type Bounds = Vec<f64>;
+
+/// Configuratoin for the statsd parser
+#[derive(Debug, Clone)]
+pub struct StatsdParseConfig {
+    /// Set specific bin masks for timeseries according to their name. The name
+    /// may be a globbing match, such like 'foo.*'. In this case all metrics
```

Comment:
Might want to say `regex match` instead of globbing here, since it's not using typical wildcard globbing semantics (afaict)